doc/author_dutka.txt

```diff
@@ -0,0 +1 @@
+I hereby place my contributions to libsc into the public domain under FreeBSD license.  Alexandre Dutka.
```

Comment:
Thanks; it's best not to mention the public domain, it's unclear how that relates to BSD in the same context.